resources/views/layouts/main.blade.php

```diff
@@ -416,6 +416,30 @@ class="nav-link @if(Request::routeIs('admin.activitylogs.*')) active @endif">
         }
     })
     @endif
+    
+const confirmDeletionForm = (e, form, text) => {
```

Comment:
can be added to resources -> js, so it can be complied using laravel mix

Reply:
Added